### PR TITLE
Adds warning for full measure directory path

### PR DESCRIPTION
**Pull request by tanushree04**
Addresses #117

### CODE REVIEW

**Review by nllong — CHANGES_REQUESTED** [review]
I think you should use the `.relative?` or `.absolute?` methods to check instead of asserting that the path exists. I think it would just be:

```
if !pn.relative?
```
or
```
if pn.absolute?
```

**Comment by tanushree04** [comment]
I don't know if it was necessary, but I also added a test to check for the warning for absolute paths

File: lib/openstudio/workflow/util/measure.rb [hunk 1]
```diff
@@ -76,6 +76,11 @@ def apply_measures(measure_type, registry, options = {}, energyplus_output_reque
 
             measure_dir_name = step.measureDirName
 
+            pn = Pathname.new(measure_dir_name)
+            if pn.absolute?
+              logger.warn "measure_dir_name should not be a full path. It should be a relative path to the measure directory or the name of the measure directory containing the measure.rb file."
```
Comment (nllong) [inline lib/openstudio/workflow/util/measure.rb:81]: 👍

File: spec/openstudio/workflow/workflow_analysis_spec.rb [hunk 2]
```diff
@@ -624,6 +624,40 @@
...
+    end
+
+    expect(osw_out).to be_instance_of Hash
+    expect(osw_out[:completed_status]).to eq 'Fail'
+    expect(osw_out[:steps]).to be_instance_of Array
+    expect(osw_out[:steps].size).to be == 1
+
+    expect(osw_out[:steps][0]).to_not be_nil
+
+    if osw_out[:steps][0][:measure_dir_name] == "/OpenStudio-workflow-gem/spec/files/full_measure_dir_osw/measures/"
```
Comment (nllong) [inline spec/openstudio/workflow/workflow_analysis_spec.rb:655]: this is fantastic. Thanks for writing the test!

**Review by nllong — APPROVED** [review]

**Comment by nllong** [comment]
@tijcolem / @tanushree04 -- do we know why the circleci is failing? Do we need circleci on this repo?

**Pull request merged**